app/views/items/index.html.erb

```diff
@@ -4,6 +4,24 @@
 <% else %>
   <h1>All Items</h1>
 <% end %>
+<table class= "Show" id= 'popularity_table'>
+  <tr>
+    <td>Most Popular Items:</td>
+    <td><%= Item.top_five.each do |item| %></td>
```

Comment:
I don't think line 10 should have an '=' in the erb tag; it might leave unexpected things in the view.